connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -499,13 +516,15 @@ ConfigDef getConfigDefFromConfigProvidingClass(String key, Class<?> cls) {
                         aliasKind + " is abstract and cannot be created. Did you mean " + childClassNames + "?";
                 throw new ConfigException(key, String.valueOf(cls), message);
             }
-            T transformation;
```

Comment:
Some basic cleanup to highlight the fact that this class is now agnostic with regards to plugin type and is not specific to SMTs.